tests/transpiler/qasm/test_qasm_to_ionq.py

```diff
@@ -67,7 +66,6 @@ def test_ionq_device_extract_gate_data():
 
     gate_data = [
         {"gate": "x", "target": 0},
-        {"gate": "not", "target": 1},
```

Comment:
IonQ supports this `not` alias, that's why we have test for it: https://docs.ionq.com/api-reference/v0.3/writing-quantum-programs#supported-gates

Reply:
https://github.com/qBraid/pyqasm/pull/86